### PR TITLE
Add new shortcode: slideshare

### PR DESCRIPTION
**Pull request by troyane**
Add new shortcode: slideshare for embeding pdfs/slides from https://www.slideshare.net/.
Update `shortcodes.md` with documentation on how to use slideshare shortcode.

### CODE REVIEW

File: exampleSite/content/docs/shortcodes.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Include content from seperate file with syntax highlighting.
 
 ### file: Parameters
 
-0 => filename [`String`] \(required\)
```
Comment (MunifTanjim) [inline exampleSite/content/docs/shortcodes.md:87]: Don't remove this spaces. Those are there for a reason. In markdown, double space at the end of line inserts a line-break.

Comment (troyane) [reply exampleSite/content/docs/shortcodes.md:87]: Sorry, it was default SublimeText behavior -- to remove trailing spaces at line end. 
Will update it shortly.

File: layouts/shortcodes/slideshare.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<iframe
+  style='width: 100%; height: 500px;'
+  frameborder='0' marginwidth='0' marginheight='0' scrolling='no'
+  src='http://www.slideshare.net/slideshow/embed_code/{{.Get "id"}}?rel=0'
```
Comment (MunifTanjim) [inline layouts/shortcodes/slideshare.html:4]: don't hardcode `http://`. Either do `https://` or remove the protocol `://`

Comment (MunifTanjim) [inline layouts/shortcodes/slideshare.html:4]: You are passing unnamed parameter. But reading named parameter "id". It won't work.

Comment (troyane) [reply layouts/shortcodes/slideshare.html]: Done, Changed to `https`

Comment (troyane) [reply layouts/shortcodes/slideshare.html]: @MunifTanjim Already done respective changes to use named parameter.

**Comment by MunifTanjim** [comment]
Thanks a lot! 😃 @troyane 🎉

**Pull request merged**